packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -72,6 +75,9 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
     this._onBarLeave = this._onBarLeave.bind(this);
     this._calloutId = getId('callout');
     this._emptyChartId = getId('_MSBC_empty');
+    this._barId = getId('_MSBC_rect_');
+    this._barIdEmptyPartToWhole = getId('_MSBC_rect_partToWhole_');
+    this._barIdEmpty = getId('_MSBC_rect_empty');
   }
 
   public render(): JSX.Element {
```

Comment:
With _isChartEmpty added, are we ever showing the placeholder bars now?

Reply:
yes, we are now showing as discussed offline